src/UI/Component/Input/Container/Filter/Factory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/* Copyright (c) 2018 Thomas Famula <famula@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Input\Container\Filter;
+
+use ILIAS\UI\Component\Signal;
+
+/**
+ * This is how a factory for filters looks like.
+ */
+interface Factory {
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Possible properties, which only Standard Filters have, should be mentioned here.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Possible rules, which only Standard Filters have, should be mentioned here.
+	 *
```

Comment:
I guess this will be added, note that this might be very short. Most is mentioned in the parent description.

Reply:
Ok, I though it would be "good practice" to have this layer for future extensibility. Even if we do not have an idea for future versions of Filters yet.

We currently see this being a problem with the Cards. We didn't have cards()->standard(). This forces us now, when introducing a second type of Card, to refactor all consumer code that used Cards before. Btw. dropdown->standard() is also an accepted thing. Not sure if we had any ideas of other future Dropdowns versions back then.

Reply:
Yeah, this is a valid point as well. My opinion to remove this is not very strong. What does @Amstutz say?

Reply:
Since we just have a Standard Filter at the moment and possible differences from future Filter member are unknown yet, I would leave the description and rules empty for now.

Reply:
No don't do that. Check the other examples, where there is just one standard field. Obviously you do not need lots of text, but please do not leave the description empty.